package.json

```diff
@@ -9,8 +9,8 @@
     "typescript": "^4.1.2"
   },
   "peerDependencies": {
-    "react": ">17",
-    "react-dom": ">17"
+    "react": "^17.0.2",
```

Comment:
should this be `>=17` instead? will this report an error with react 18 since it's a different major version?

Reply:
also why do we enforce >=17 instead of >=16? is there another dependency we have that requires it?

Reply:
good point, i was just reverting to what we had before. let me try and test the package on 16 and 18.